MajorNormalization.md

```diff
@@ -55,10 +57,10 @@ Sample Response
 Response Information
 -----
 
-The response returns a single data node which contains a list of normalized majors. These normalized majors are ordered by the confidence score. Confidence scores range from 0.0 to 1.0. 
+The response returns a single data node which contains a list of normalized majors and the current data version. These normalized majors are ordered by the confidence score. Confidence scores range from [0, 1].
 
 Versioning
 -----------
-The response from the Major Normalization Service is versioned with the current version being 1.0. The data set used to perform the normalization is unversioned.
+The data that backs the Major Normalization Service is versioned. The data version is returned at the "data" node level.
```

Comment:
The second sentence here is a bit confusing to me. I would just call out the node that contains the version info. like, "The data that backs the Major Normalization Service is versioned. The current version is returned in the `data_version` field of the response."